src/ResourceManager/Compute/Commands.Compute/VirtualMachine/Action/SetAzureVMCommand.cs

```diff
@@ -70,34 +70,29 @@ public class SetAzureVMCommand : VirtualMachineBaseCmdlet
 
         [Parameter(
             Mandatory = true,
-            Position = 2,
```

Comment:
@markcowl we should let Compute know about these changes (both to `SwitchParameter` properties and the addition of `AsJob`) so they can update their generator accordingly

Reply:
We have let them know, but yes, we should let them know explicitly how to use this - meanwhile they have told us to edit the generated code in these cases.